### DISCUSSION TITLE
getting "Unknown desc = OK: HTTP status code 200; transport: missing content-type field"

### DISCUSSION THREAD

**Discussion by sumit-joshi-mt** [linkerd/linkerd2#5428] [Q&A]
Hi, 
We are getting "rpc error: code = Unknown desc = OK: HTTP status code 200; transport: missing content-type field" error intermittently after adding linkerd to service; destination was up and running and request was not reached to destination as well.
has anyone seen this? 
We are using proxy version ```edge-20.10.6``` on kubernetes ```v1.17.12```

**Comment by kleimkuhler** [comment]
Hi @sumit-joshi-mt. Is this a gRPC service by chance?

Also I would recommend upgrading to either the latest edge (`edge-20.12.4`) or the latest stable (`stable-2.9.1`)—both of which are later release than `edge-20.10.6`. Once you do that it will be helpful to know if the problem persists and we can go from there.

**Reply by sumit-joshi-mt** [reply]
Hi, @kleimkuhler Yes. it is grpc service. Okay will try with latest release in that case. 
Thank you

**Comment by cpretzer** [comment]
hi @sumit-joshi-mt how did your tests with the latest release go? Did it resolve the issue?
